cmd/hubproxy/main.go

```diff
@@ -189,17 +189,22 @@ func run() error {
 	// Initialize storage if DB URI is provided
 	var store storage.Storage
 	dbURI := viper.GetString("db")
```

Comment:
Let's just set the defalut value to `"sqlite::memory:"` at the viper/cobra level.